x/staking/types/errors.go

```diff
@@ -50,4 +50,5 @@ var (
 	ErrNoHistoricalInfo                = sdkerrors.Register(ModuleName, 38, "no historical info found")
 	ErrEmptyValidatorPubKey            = sdkerrors.Register(ModuleName, 39, "empty validator public key")
 	ErrUnbondingNotFound               = sdkerrors.Register(ModuleName, 40, "unbonding operation not found")
+	ErrUnbondingOnHoldRefCountNegative = sdkerrors.Register(ModuleName, 41, "cannot un-hold unbonding operation that is not on hold")
```

Comment:
Think this is a panic situation, not an error, right? Errors are for recoverable, possible things but this is something that would imply a bug in our code and isn't recoverable.

Reply:
see https://github.com/cosmos/cosmos-sdk/pull/12796#discussion_r938524287